docs-source/content/faq/resource-settings.md

```diff
@@ -0,0 +1,162 @@
+
+- **Best Effort QoS** (lowest priority): If you don't configure requests and limits for a pod, then the pod is given a `best-effort` QoS. In cases where a node runs out of non-shareable resources, a Kubernetes `kubelet` default out-of-resource eviction policy evicts running pods with the `best-effort` QoS first.
+
+- **Burstable QoS** (medium priority): If you configure both resource requests and limits for a pod, and set the requests to be less than their respective limits, then the pod will be given a `burstable` QoS. Similarly, if you only configure resource requests (without limits) for a pod, then the pod QoS is also `burstable`. If a node runs out of non-shareable resources, the node's `kubelet` will evict `burstable` pods only when there are no more running `best-effort` pods.
+
+- **Guaranteed QoS** (highest priority): If you set a pod's requests and the limits to equal values, then the pod will have a `guaranteed` QoS. These settings indicates that your pod will consume a fixed amount of memory and CPU. With this configuration, if a node runs out of shareable resources, then a Kubernetes node's `kubelet` will evict `best-effort` and `burstable` QoS pods before terminating `guaranteed` QoS pods.
+
+{{% notice note %}} 
+For most use cases, Oracle recommends configuring WebLogic pods with memory and CPU requests and limits, and furthermore setting requests equal to their respective limits in order to ensure a `guaranteed` QoS.
+{{% /notice %}}
+
+In newer version of Kubernetes, it is possible to fine tune scheduling and eviction policies using [Pod Priority Preemption](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/). TBD Ryan - is it possible to change the priority class of a pod?
+
+### Java heap size and memory resource considerations
+
+{{% notice note %}} 
+For most use cases, Oracle recommends configuring Java heap sizes for WebLogic pods instead of relying on defaults.
+{{% /notice %}}
+
+#### Overview
+
+It's extremely important to set correct heap sizes, memory requests, and memory limits for WebLogic JVMs and Pods. 
+
+A WebLogic JVM heap must be sufficiently sized to run its applications and services, but should not be sized too large so as not to waste memory resources.
+
+A pod memory limit must be sufficiently sized to accommodate the configured heap (and off-heap) requirements, but  not too big to waste memory resources. If a JVM's memory usage (sum of heap and native memory) exceeds its pod's limit, then the JVM process will be abruptly killed due to an out-of-memory error and the WebLogic container will consequently automatically restart due to a liveness probe failure.  
+
+{{% notice warning %}}
+If resource requests and resource limits are set too high, then your pods may not be scheduled due to lack of node resources, will unnecessarily use up CPU shared resources that could be used by other pods, or may prevent other pods from running.
+{{% /notice %}}
+
+#### Default heap sizes
```

Comment:
I think the topic here is "Importance of setting heap sizes"

Reply:
Or, put another way, we should end this section more strongly to say that customers must set the heap min and max (or the maximum heap percentage) and at least container memory request.

Reply:
Changed the title and added recommendation for setting heap min and max and container memory request.